sentences/ro/_common.yaml

```
@@ -54,7 +54,7 @@ expansion_rules:
   este: "(e | este)"
   modifica: "(schimb(ă|a) | modific(ă|a) | seteaz(ă|a))"
   in: "((î|i)n)"
-  din: "(din | (î|i)n | pentru | de la)"
+  din: "(din | in | în | pentru | [de] [la] | [de] [pe] )"
```

```suggestion
  din: "(din | (î|i)n | pentru | [de] la | [de] pe | de )"
```

Nu putem avea toate cuvintele dintr-o sintagma optionale